packages/ketcher-react/src/script/ui/views/toolbars/FloatingTools/FloatingTools.container.ts

```diff
@@ -0,0 +1,32 @@
+import { connect } from 'react-redux'
+import { Dispatch } from 'redux'
+import {
+  getFloatingToolsHandlePos,
+  getFloatingToolsVisible
+} from 'src/script/ui/state/floatingTools/selectors'
+import { onAction } from '../../../state'
+import {
+  FloatingTools,
+  type FloatingToolsCallProps,
+  type FloatingToolsProps
+} from './FloatingTools'
+
+const mapStateToProps = (state: any): FloatingToolsProps => {
+  return {
+    visible: getFloatingToolsVisible(state),
+    HandlePos: getFloatingToolsHandlePos(state),
```

Comment:
curious why it is capitalised? Looks like it is not a common convention, any reason?

Reply:
My bad, this is a typo. Thank you so much!